python_modules/dagster/dagster/_core/pipes/utils.py

```diff
@@ -280,15 +316,30 @@ def get_params(self) -> Iterator[PipesParams]:
     @abstractmethod
     def download_messages_chunk(self, index: int, params: PipesParams) -> Optional[str]: ...
 
-    def _reader_thread(
-        self, handler: "PipesMessageHandler", params: PipesParams, is_task_complete: Event
+    def download_stdout_chunk(self, params: PipesParams) -> Optional[str]:
+        raise NotImplementedError()
```

Comment:
`NotImplementedError` instead of abstract methods because we don't need want to require every `PipesBlobStoreMessageReader` to implement these.